src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -1,29 +0,0 @@
-<Project>
```

Comment:
Either we need to remove this from Universe's artifact.props https://github.com/aspnet/Universe/blob/a01aee7e9c92b64017c71c740668ab688c42a26f/build/artifacts.props#L12 or we need to keep this. I'd imagine this would fail the "coherence" check.

Reply:
I'll update it before I merge this.